packages/mdx-components/README.md

```diff
@@ -0,0 +1,3 @@
+# @carbon-platform/mdx-components
+
+Docs are located [here](../../docs/packages-mdx-components.md).
```

Comment:
should we create these docs even if empty? or add a TODO/create a ticket for this?

Reply:
Yep, I have a todo to add that in this PR. Probably working on that this afternoon.